src/components/VoiceCompair/VoiceCompair.js

```diff
@@ -211,8 +213,34 @@ const VoiceCompair = props => {
 
         let word_result = (result_per_words == 100) ? "correct" : "incorrect";
 
+        const client = new S3Client({
+          region: process.env.REACT_APP_AWS_s3_region,
```

Comment:
what if - we no need to push this data to s3?

Reply:
Added condition in new commit with new env variable